### DISCUSSION TITLE
Portable installation

### DISCUSSION THREAD

**Discussion by YSCohen** [ScoopInstaller/Scoop#5385] [Ideas]
I have scoop installed in my home folder, and when I tried doing a second install on a USB drive, it complained that scoop is already installed. I know that I can trick it into letting me double-install, but maybe there should be an option to create a specifically portable installation that would, e.g., make the shims relative paths (is that possible?), prevent any changes to env vars, etc.
just a thought

**Comment by rashil2000** [comment]
There has been some discussion on this:
- https://github.com/ScoopInstaller/Scoop/pull/5369
- https://github.com/ScoopInstaller/Scoop/issues/3915
